astropy/coordinates/representation.py

```diff
@@ -20,23 +20,29 @@
 from ..extern import six
 from ..utils import ShapedLikeNDArray, classproperty
 from ..utils.compat import NUMPY_LT_1_12
-from ..utils.compat.numpy import broadcast_arrays
-
-__all__ = ["BaseRepresentation", "CartesianRepresentation",
-           "SphericalRepresentation", "UnitSphericalRepresentation",
-           "PhysicsSphericalRepresentation", "CylindricalRepresentation"]
-
-# Module-level dict mapping representation string alias names to class.
-# This is populated by the metaclass init so all representation classes
-# get registered automatically.
+from ..utils.compat.numpy import broadcast_arrays, broadcast_to
+
+__all__ = ["RepresentationBase", "BaseRepresentation",
```

Comment:
OK, the name is indeed bad. Not too fond of the very long name, but cannot think of anything better, so let's just go with it.